To: vim_dev@googlegroups.com
Subject: Patch 7.3.128
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.128
Problem:    Another compiler warning for signed pointer.
Solution:   Use unsigned int argument for sscanf().
Files:	    src/mark.c


*** ../vim-7.3.127/src/mark.c	2010-08-15 21:57:25.000000000 +0200
--- src/mark.c	2010-12-17 20:00:05.000000000 +0100
***************
*** 1750,1756 ****
  	    {
  		if (line[1] != NUL)
  		{
! 		    sscanf((char *)line + 2, "%ld %u", &pos.lnum, &pos.col);
  		    switch (line[1])
  		    {
  			case '"': curbuf->b_last_cursor = pos; break;
--- 1750,1759 ----
  	    {
  		if (line[1] != NUL)
  		{
! 		    unsigned u;
! 
! 		    sscanf((char *)line + 2, "%ld %u", &pos.lnum, &u);
! 		    pos.col = u;
  		    switch (line[1])
  		    {
  			case '"': curbuf->b_last_cursor = pos; break;
*** ../vim-7.3.127/src/version.c	2011-02-25 14:46:06.000000000 +0100
--- src/version.c	2011-02-25 15:10:34.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     128,
  /**/

-- 
FATHER:    Who are you?
PRINCE:    I'm ... your son ...
FATHER:    Not you.
LAUNCELOT: I'm ... er ... Sir Launcelot, sir.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
